pom.xml

```diff
@@ -118,7 +118,7 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <version.spring.boot>2.7.10</version.spring.boot>
         <version.guava>31.1-jre</version.guava>
-        <version.spring.cloud>2021.0.5</version.spring.cloud>
```

Comment:
In `spring-cloud-azure-5.x` profile, `version.spring.cloud` is already `2022.0.2`.

Reply:
Closing this PR.